colaraz/lms/templates/main.html

```diff
@@ -446,9 +446,14 @@ <h2 class="account-status-title sr">${_("Account Status Info")}: </h2>
 
     <%include file="widgets/segment-io-footer.html" />
     <script type="text/javascript">
+        // Colaraz Notifications API variables
         var colarazNotificationsFetchingUrl = "${reverse('colaraz_features:notifications_handler', kwargs={'api_method': 'fetch'})}";
         var colarazNotificationsMarkingUrl = "${reverse('colaraz_features:notifications_handler', kwargs={'api_method': 'mark'})}";
-        var notificationsRefreshTime = ${ getattr(settings, 'COLARAZ_NOTIFICATIONS', {}).get('REFRESH_TIME_MILLISECONDS', 30000)};
+        var colarazNotificationsRefreshTime = ${ getattr(settings, 'COLARAZ_NOTIFICATIONS', {}).get('REFRESH_TIME_MILLISECONDS', 30000)};
```

Comment:
@HamzaIbnFarooq 
Just for your knowledge that this line of code is not behaving the way you are expecting. For example, if we don't configure `COLARAZ_NOTIFICATIONS` in the settings, this line will crash the system.
![image](https://user-images.githubusercontent.com/42294172/83227419-9cc03100-a19d-11ea-9b4d-349ffc9b3801.png)

```
File "/edx/app/edxapp/venvs/edxapp/local/lib/python2.7/site-packages/mako/template.py", line 454, in render_unicode
    as_unicode=True)
  File "/edx/app/edxapp/venvs/edxapp/local/lib/python2.7/site-packages/mako/runtime.py", line 829, in _render
    **_kwargs_for_callable(callable_, data))
  File "/edx/app/edxapp/venvs/edxapp/local/lib/python2.7/site-packages/mako/runtime.py", line 864, in _render_context
    _exec_template(inherit, lclcontext, args=args, kwargs=kwargs)
  File "/edx/app/edxapp/venvs/edxapp/local/lib/python2.7/site-packages/mako/runtime.py", line 890, in _exec_template
    callable_(context, *args, **kwargs)
  File "/tmp/mako_lms/618647d7cc5e9d2f629dc617e0c7244f/colaraz/lms/templates/main.html.py", line 529, in render_body
    __M_writer(filters.decode.utf8( getattr(settings, 'COLARAZ_NOTIFICATIONS', {}).get('REFRESH_TIME_MILLISECONDS', 30000)))
AttributeError: 'NoneType' object has no attribute 'get'
```

Reply:
I have added a default value in the edx-platform to handle this.